app/scripts/controllers/preferences.js

```diff
@@ -671,7 +671,7 @@ class PreferencesController {
   /**
    * A getter for `tokens` and `accountTokens` related states.
    *
-   * @param {string} selectedAddress A new hex address for an account
+   * @param {string=} selectedAddress A new hex address for an account
```

Comment:
Interesting - you prefer the Closure syntax? The JSDoc syntax is more common in this repo:
```suggestion
   * @param {string} [selectedAddress] A new hex address for an account
```

Reply:
Nah, this is just a random decision made in the moment—I guess I know Closure syntax off-hand better than the JSDoc syntax.